src/BloomExe/XmlHtmlConverter.cs

```diff
@@ -40,15 +34,10 @@ public static XmlDocument GetXmlDomFromHtmlFile(string path, bool includeXmlDecl
 		public static XmlDocument GetXmlDomFromHtml(string content, bool includeXmlDeclaration = false)
 		{
 			var dom = new XmlDocument();
-			//hack. tidy deletes <span data-libray='somethingImportant'></span>
-			// and also (sometimes...apparently only the first child in a parent) <i some-important-attributes></i>
-			content = content.Replace("></span>", ">REMOVEME</span>");
-			content = content.Replace("></i>", ">REMOVEME</i>");
+			content = AddFillerToKeepTidyFromRemovingEmptyElements(content);
 
 			// It also likes to insert newlines before <b>, <u>, and <i>, and convert any existing whitespace
 			// there to a space.
-			// It probably does the same to embedded <span> elements, too, so we might need a similar fix
-			// for those if we start using them in persistent data.
 			content = content.Replace(@"<b>", "REMOVEWHITESPACE<b>");
 			content = content.Replace(@"<i>", "REMOVEWHITESPACE<i>");
 			content = content.Replace(@"<u>", "REMOVEWHITESPACE<u>");
```

Comment:
I'm surprised you didn't move this filler-insertion into the new method, too. Does the other caller of AddFillerToKeepTidyFromRemovingEmptyElements not need it?